framework/doc/content/source/materials/ReporterOffsetFunctionMaterial.md

```diff
@@ -0,0 +1,21 @@
+# ReporterOffsetFunctionMaterial
+
+!syntax description /Materials/ReporterOffsetFunctionMaterial
+
+This creates a material that is the sum of a function shifted by a set of points contained in a reporter.  The reporter can contain points by specifying [!param](/Materials/ReporterOffsetFunctionMaterial/point_name) or by three seperate reporters containing the (x,y,z) coordinates of the points using [!param](/Materials/ReporterOffsetFunctionMaterial/x_coord_name), [!param](/Materials/ReporterOffsetFunctionMaterial/y_coord_name), and [!param](/Materials/ReporterOffsetFunctionMaterial/z_coord_name). This can be useful for creating a field containing multiple sources, as shown in [offset].
```

Comment:
```suggestion
This creates a material that is the sum of functions shifted by a set of points contained in a reporter. The reporter can contain points by specifying [!param](/Materials/ReporterOffsetFunctionMaterial/point_name) or by three separate reporters containing the (x,y,z) coordinates of the points using [!param](/Materials/ReporterOffsetFunctionMaterial/x_coord_name), [!param](/Materials/ReporterOffsetFunctionMaterial/y_coord_name), and [!param](/Materials/ReporterOffsetFunctionMaterial/z_coord_name). This can be useful for creating a field containing multiple sources, as shown in [offset].
```

Reply:
This first sentence is redundant with the class description. I would delete and focus this paragraph on how to specify the points 

Add the equation for the sum so people can see the minus